encoder/vaapiencoder_vp8.cpp

```diff
@@ -75,6 +75,10 @@ void VaapiEncoderVP8::resetParams()
 {
     //5 times compress ratio
     m_maxCodedbufSize = width() * height() * 3 / 2;
+    //when the resolution of video is very small such as 96x96 and buffer plus the header of vp8,
+    //the size of actual output buffer maybe be bigger the m_maxCodedbufSize.
+    //So plusing 4096 avoids the situation.
+    m_maxCodedbufSize += 0x1000;
```

Comment:
No it's not driver's issue. The root cause of this problem is, when you do encode, it will encode frame to slice data,then append a header to the slice data. When resolution is small, the header + slice data may excess the size of width*height*3/2. And 4k is enough for any headers.So linhai provide this solution.